README.md

```diff
@@ -185,29 +185,36 @@ This repository contains a curated list of awesome open source libraries that wi
 * [LightLLM](https://github.com/ModelTC/lightllm) ![](https://img.shields.io/github/stars/ModelTC/lightllm.svg?style=social) - LightLLM is a Python-based LLM (Large Language Model) inference and serving framework, notable for its lightweight design, easy scalability, and high-speed performance.
 * [LLMonitor](https://github.com/LLMonitor/llmonitor) ![](https://img.shields.io/github/stars/LLMonitor/llmonitor.svg?style=social) - Observability & analytics for AI apps and agents.
 * [LocalAI](https://github.com/mudler/LocalAI) ![](https://img.shields.io/github/stars/mudler/LocalAI.svg?style=social) - LocalAI is a drop-in replacement REST API that's compatible with OpenAI API specifications for local inferencing.
+* [lunary](https://github.com/lunary-ai/lunary) ![](https://img.shields.io/github/stars/lunary-ai/lunary.svg?style=social) - Open-source observability & prompt platform for LLMs.
 * [m2cgen](https://github.com/BayesWitnesses/m2cgen) ![](https://img.shields.io/github/stars/BayesWitnesses/m2cgen.svg?style=social) - A lightweight library which allows to transpile trained classic machine learning models into a native code of C, Java, Go, R, PHP, Dart, Haskell, Rust and many other programming languages.
 * [MLEM](https://github.com/iterative/mlem/) ![](https://img.shields.io/github/stars/iterative/mlem.svg?style=social) - Version and deploy your ML models following GitOps principles.
 * [MLServer](https://github.com/SeldonIO/mlserver) ![](https://img.shields.io/github/stars/SeldonIO/mlserver.svg?style=social) - An inference server for your machine learning models, including support for multiple frameworks, multi-model serving and more.
 * [mltrace](https://github.com/loglabs/mltrace) ![](https://img.shields.io/github/stars/loglabs/mltrace.svg?style=social) - a lightweight, open-source Python tool to get "bolt-on" observability in ML pipelines.
 * [MLWatcher](https://github.com/anodot/MLWatcher) ![](https://img.shields.io/github/stars/anodot/MLWatcher.svg?style=social) - MLWatcher is a python agent that records a large variety of time-serie metrics of your running ML classification algorithm. It enables you to monitor in real time.
 * [Model Server for Apache MXNet (MMS)](https://github.com/awslabs/mxnet-model-server) ![](https://img.shields.io/github/stars/awslabs/mxnet-model-server.svg?style=social) - A model server for Apache MXNet from Amazon Web Services that is able to run MXNet models as well as Gluon models (Amazon's SageMaker runs a custom version of MMS under the hood).
+* [ModelCache](https://github.com/codefuse-ai/ModelCache) ![](https://img.shields.io/github/stars/codefuse-ai/ModelCache.svg?style=social) - ModelCache is a LLM semantic caching system aiming to enhance user experience by reducing response time via cached query-result pairs.
```

Comment:
Let's remove this LLM-cache-management one to merge this PR, and discuss as part of our sync